cc/concurrent_selfplay.cc

```diff
@@ -129,7 +134,7 @@ DEFINE_uint64(seed, 0,
 DEFINE_double(resign_threshold, -0.999, "Resign threshold.");
 DEFINE_double(disable_resign_pct, 0.1,
               "Fraction of games to disable resignation for.");
-DEFINE_int32(num_games, 1, "Total number of games to play.");
+DEFINE_int32(num_games, 0, "Total number of games to play.");
```

Comment:
maybe expand the docstring here -- does zero have a special meaning?  i.e., 0 = run continuous?  or is this a safety (i.e. "hey you didn't set it so we quit").

Reply:
The previous value of `1` was a bug: `num_games` and `run_forever` are supposed to be mutually exclusive.